[2024-04-05 17:36:55,698][flwr][INFO] - Starting Flower simulation, config: ServerConfig(num_rounds=10, round_timeout=None)
[2024-04-05 17:36:59,404][flwr][INFO] - Flower VCE: Ray initialized with resources: {'GPU': 1.0, 'memory': 76477845709.0, 'CPU': 32.0, 'node:127.0.0.1': 1.0, 'object_store_memory': 37061933875.0}
[2024-04-05 17:36:59,408][flwr][INFO] - Initializing global parameters
[2024-04-05 17:36:59,409][flwr][INFO] - Requesting initial parameters from one random client
[2024-04-05 17:37:01,979][flwr][ERROR] - [36mray::launch_and_get_parameters()[39m (pid=27448, ip=127.0.0.1)
  File "python\ray\_raylet.pyx", line 640, in ray._raylet.execute_task
  File "python\ray\_raylet.pyx", line 644, in ray._raylet.execute_task
  File "C:\Users\Admin\anaconda3\envs\flwrpytorch\lib\site-packages\flwr\simulation\ray_transport\ray_client_proxy.py", line 135, in launch_and_get_parameters
    client: Client = _create_client(client_fn, cid)
  File "C:\Users\Admin\anaconda3\envs\flwrpytorch\lib\site-packages\flwr\simulation\ray_transport\ray_client_proxy.py", line 168, in _create_client
    client_like: ClientLike = client_fn(cid)
  File "c:\Users\Admin\Desktop\Vipul\Part-II\client.py", line 74, in client_fn
    return FlowerClient(
AttributeError: 'FlowerClient' object has no attribute 'to_client'
